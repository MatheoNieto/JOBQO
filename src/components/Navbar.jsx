import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';

const useStyles = makeStyles(() => ({
  title: {
    flexGrow: 1,
  },
}));

const Navbar = ({ children }) => {
  const classes = useStyles();

  return (
    <>
      <div className={classes.root}>
        <AppBar position='static'>
          <Toolbar>
            <Typography
              variant='h4'
              className={classes.title}
              align='center'
            >
              JOBQO
            </Typography>
          </Toolbar>
        </AppBar>
        <Toolbar />

        {children}
      </div>
    </>
  );
};

export default Navbar;
