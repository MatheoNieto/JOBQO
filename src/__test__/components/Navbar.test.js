import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';
import Navbar from '../../components/Navbar';

describe('<Navbar />', () => {
  test('Render del componente Navbar', () => {
    const navbar = mount(<Navbar />);
    expect(navbar.length).toEqual(1);;
  });
});

describe('Navbar Snapshot', () => {
  test('Comprobar la UI del componente Navbar', () => {
    const navbar = create(<Navbar />);
    expect(navbar.toJSON()).toMatchSnapshot();
  });
});
