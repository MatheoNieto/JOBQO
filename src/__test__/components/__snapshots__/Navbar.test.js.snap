// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Snapshot Comprobar la UI del componente Navbar 1`] = `
<div>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h4
        className="MuiTypography-root makeStyles-title-1 MuiTypography-h4 MuiTypography-alignCenter"
      >
        JOBQO
      </h4>
    </div>
  </header>
  <div
    className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
  />
</div>
`;
